RELEASES.md

```diff
@@ -0,0 +1,77 @@
+# Grafana Releases
+
```

Comment:
Add an intro that tells the reader what this document will tell them. Pretend you are explaining it to a brand new employee.

And as a general practice, avoid stacked headings. Either add an intro or remove one of the headings.